Remotion/Web/Development.WebTesting.IntegrationTests/TestExecutionScreenshotRecorderTest.cs

```diff
@@ -43,12 +44,15 @@ public void SetUp ()
     [TearDown]
     public void TearDown ()
     {
-      var files = Directory.GetFiles (_tempSavePath, "*.png");
+      if (Directory.Exists (_tempSavePath))

```

Comment:
As we spoke, this line needed to be here all along but technically has nothing to do with this PR. We probably don't need an extra Jira issue for this and it may not even need its own commit. I'm not sure.

Reply:
The entire change in this method should be an extra commit, first. yes, issue since it is a problem in the test and not just cosmetics and should be backported/tracked.

Reply:
Tracking in [RM-7280](https://re-motion.atlassian.net/browse/RM-7280). Added https://github.com/re-motion/Trunk-SVN2Git/pull/26.